src/app/pages/apps/components/installed-apps/app-settings-button/app-settings-button.component.html

```diff
@@ -29,9 +29,24 @@
       {{ 'Unset Pool' | translate }}
     </button>
   }
-  <button mat-menu-item ixTest="container-images" [routerLink]="['/apps', 'manage-container-images']">
+
+  <button
+    mat-menu-item
+    ixTest="container-images"
+    [routerLink]="['/apps', 'manage-container-images']"
+  >
     {{ 'Manage Container Images' | translate }}
   </button>
+
+  <button
```

Comment:
Turns out this menu item requires Pool to be selected first. Let's disable it, until pool is selected.